src/services/custom/index.js

```diff
@@ -0,0 +1,30 @@
+import * as cache from '../cache';
```

Comment:
This file seems like a confusing thing to modularize, I might just not be seeing where you're headed though.